numba/core/event.py

```diff
@@ -423,3 +429,53 @@ def on_exit(*exc_details):
 
         start_event(kind, data=data)
         yield
+
+
+def _get_native_ident():
+    try:
+        return threading.get_native_ident()
+    except AttributeError:
+        # Fallback for python <3.8
+        return threading.get_ident()
+
+
+def _prepare_chrome_trace_data(listener: RecordingListener):
+    """Prepare events in `listener` for serializing as chrome trace data.
+    """
+    # The spec for the trace event format can be found at:
+    # https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/edit   # noqa
```

Comment:
Is there a non-google doc version of the spec?

Reply:
I tried to find it but no. All other tools I found are referring to it via the google doc link. Google tends to use google doc for spec.

Reply:
ok, thanks for checking.